fuzzbench/run_experiment.py

```diff
@@ -18,20 +18,29 @@
 import redis
 import rq
 
+from common import config_utils, environment, yaml_utils
+from experiment.build import docker_images
 from fuzzbench import jobs
 
 
-def run_experiment():
+def run_experiment(config):
     """Main experiment logic."""
     print('Initializing the job queue.')
     # Create the queue for scheduling build jobs and run jobs.
     queue = rq.Queue('build_n_run_queue')
+
+    images_to_build = docker_images.get_images_to_build(config['fuzzers'],
+                                                        config['benchmarks'])
     jobs_list = []
-    jobs_list.append(
-        queue.enqueue(jobs.build_image,
-                      'base-image',
-                      job_timeout=600,
-                      job_id='base-image'))
+    for name, obj in images_to_build.items():
+        depends = obj.get('depends_on', None)
+        jobs_list.append(
+            queue.enqueue(jobs.build_image,
+                          image=obj,
+                          job_timeout=1800,
+                          result_ttl=-1,
+                          job_id=name,
+                          depends_on=None if not depends else depends[0]))
```

Comment:
consider: `depends[0] if `depends_on` in image else None`

Reply:
Sure. Done.

Reply:
Sg!